tests/plugins/pastefromwordimage/helpers.js

```diff
@@ -0,0 +1,112 @@
+/* bender-tags: editor,pastefromwordimage */
+/* bender-ckeditor-plugins: pastefromwordimage,ajax */
+
+( function() {
+	'use strict';
+
+	bender.test( {
+		cacheBuster: function() {
+			return ( new Date() ).getTime().toString() + Math.floor( Math.random() * 1000000 ).toString();
+		},
+
+		'test img tags extraction from html string': function() {
+			var sourceLength = 5,
+				i;
+
+			for ( i = 0; i < sourceLength; i++ ) {
+				bender.tools.testInputOut( 'source' + i, function( input, output ) {
+					var expectedResult = JSON.parse( output.replace( /<!--/, '' ).replace( /-->/, '' ) ),
+						actualResult = CKEDITOR.plugins.pastefromwordimage.extractImgTagsFromHtml( input ),
+						j;
+
+					assert.areSame( expectedResult.length, actualResult.length );
+					for ( j = 0; j < actualResult.length; j++ ) {
+						assert.areSame( expectedResult[ j ], actualResult[ j ] );
+					}
+				} );
+			}
+		},
+
+		'test extract images hexstrings from rtf': function() {
+			var oldEnv = CKEDITOR.env.mac,
+				actualResult,
+				testCase,
+				// Test description:
+				// 1. Single Image
+				// 2. Mixed: image online, image offline, word shape
+				// 3. No images in RTF
+				// 4. Image with hyperlink
+				// 5. Image with changed wrapping option
+				testCases = [
+				{
```

Comment:
Wrong indentation, it should be intended by one tab more (following elements too), just like it is in the "test extract images hexstrings from rtf for MacOS" TC.

Reply:
And here we come to resource loading. Please change it to load resources asynchronously (test of will have to work asynchronously too, so it will use `wait` and `resume`).

Still `CKEDITOR.ajax.load` can be used for async loading.